backend/audit/cross_validation/additional_ueis.py

```diff
@@ -0,0 +1,49 @@
+def additional_ueis(sac_dict):
+    """
+    Checks that there are additional UEIs if
+    general_information.multiple_ueis_covered is True, and that there are
+    none if it's not True.
+    """
+    all_sections = sac_dict["sf_sac_sections"]
+    addl_ueis_checked = all_sections["general_information"].get("multiple_ueis_covered")
+    auditee_uei = all_sections["general_information"].get("auditee_uei")
+    addl_ueis = all_sections["additional_ueis"]
+    if addl_ueis_checked:
+        """
+        We need to check that:
+
+        +   all_sections.additional_ueis isn't an empty list
+        +   The first object in it has a value for additional_uei
+        +   None of the values are the same as auditee_uei
+        """
+        if not addl_ueis:
+            msg = "".join(
```

Comment:
**question (non-blocking)**: This is a reasonable solution for formatting long strings, but I wonder how it differs from the possibly more idiomatic

```
("general_information.multiple_ueis_covered is checked, "
"but no additional UEIs were found.")
```

which accomplishes the same thing without the explicit `"".join(`?